### DISCUSSION TITLE
Is multi-dimensional performance something relevant to matchup adjustments?

### DISCUSSION THREAD

**Discussion by justanediblefriend** [OpenDebates/openskill.py#142] [Q&A] [answered]
Try to wrap its mind around "By introducing this multidimensional aspect, the Plackett-Luce model provides a richer framework for ranking players based on their abilities in various dimensions." It's not obvious what in the model does this or what this means to it, is this something that would allow for you to account for matchups?

Like say you have a bunch of combatants who can use multiple weapons and you want to see how powerful each weapon is, so you give every weapon a rating. But sometimes, a very powerful weapon loses out to someone wielding a weaker weapon, or sometimes a very powerful weapon is only powerful when it's combined with another weaker weapon. The description PlackettLuce model makes it sound like this model can adjust the probability of victory on the basis of factors like this, but nothing else seems to suggest that this is true, so thought it might be good to ask.

**Comment by vivekjoshy** [comment, marked as answer]
That description is of the original Plackett-Luce model introduced by Plackett and Luce. PlackettLuce is capable of handling things like Item-Covariates using an algorithm called [PLADMM](https://cran.r-project.org/web/packages/PlackettLuce/vignettes/PLADMM.html). It's been done before, just never for rating systems. And of course, implementing it for rating systems is currently an open problem.
